Classes/EventListener/EventListenerManager.php

```diff
@@ -24,7 +28,7 @@
  *
  * @Flow\Scope("singleton")
  */
-class EventListenerLocator
+class EventListenerManager
```

Comment:
I actually think `Locator` was a more expressive name to what this is/was about. `Manager` is rather arbitrary and the need to change probably came from adding the responsibility to `catch up`. But this breaks SRP, so maybe we should just move that to a more concrete service, like `EventListenerConsistencyManager`?